src/test/java/org/folio/auth/authtokenmodule/AuthTokenTest.java

```diff
@@ -131,14 +127,13 @@ public static void setUpClass(TestContext context) throws NoSuchAlgorithmExcepti
           if (mainRes.failed()) {
             context.fail(mainRes.cause());
           } else {
+            var tenantAttributes = new JsonObject().put("module_to", "mod-authtoken-1.0.0");
+            initializeTenantForTokenStore(tenant, tenantAttributes);
             async.complete();
```

Comment:
This should be futurized to avoid these nested if-clauses.

```
vertx.deployVerticle(...)
.compose(x -> vertx.deployVerticle(...))
.onComplete(context.asyncAssertSuccess(x -> {
  var tenantAttributes = new JsonObject().put("module_to", "mod-authtoken-1.0.0");
  initializeTenantForTokenStore(tenant, tenantAttributes);
}));
```

The logger.info is not needed.

Reply:
Done